### PR TITLE
PROD-1397 react-date-time-group does not work with webpack

### PR DESCRIPTION
**Pull request by hx-markterry**
https://holidayextras.jira.com/browse/PROD-1397

#### What does this PR do? (please provide any background)

This work updates the `prepublish` npm script to add the required plugins and presets so this component can be consumed by a client side project.

#### What tests does this PR have?

No extra tests, tool update only.

#### How can this be tested?

Checkout the **tripapplite** branch of `carHire-demo`.

    npm install
    cd node_modules/tripapp-product-carhire/node_modules/react-date-time-group
    npm install
    npm run prepublish
    cd ../../../..

Then run tripapp as usual, it should build

#### Screenshots / Screencast

Example car hire availability URL:

[http://localhost:5000/?pickUpLocation=&dropOffLocation=&Date=2016-03-01&StartTime=00%3A30&Date=2016-03-01&EndTime=00%3A30&dateOfBirth=#/carhire?from=2016-04-10 12%3A30&to=2016-04-15 13%3A30&date_of_birth=1981-01-01&agent=web1&pickup_location=FL1757&dropoff_location=FL1757](http://localhost:5000/?pickUpLocation=&dropOffLocation=&Date=2016-03-01&StartTime=00%3A30&Date=2016-03-01&EndTime=00%3A30&dateOfBirth=#/carhire?from=2016-04-10 12%3A30&to=2016-04-15 13%3A30&date_of_birth=1981-01-01&agent=web1&pickup_location=FL1757&dropoff_location=FL1757)

Car hire engine URL:

[http://localhost:5000/#/carhire/engine](http://localhost:5000/#/carhire/engine)

#### What gif best describes how you feel about this work?

giphy

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by GeorgeF72** [comment]
:one:

**Comment by GeorgeF72** [comment]
:+1:

**Comment by lukehansell** [comment]
taking :two:

File: package.json [hunk 1]
```diff
@@ -1,22 +1,17 @@
 {
   "name": "react-date-time-group",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "description": "Datepicker with optional time",
   "main": "dist/DateTimeGroup.js",
   "scripts": {
     "ci": "npm run lint && npm run coverage",
-    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage",
+    "coverage": "babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- --require react-tests-globals-setup test/test* && istanbul check-coverage",
```
Comment (lukehansell) [inline package.json]: are we sure we want to use babel-node over `babel/register` compiler?

**Comment by lukehansell** [comment]
looks to have resolved the issue with it not building :+1:

**Pull request merged**